recipes/dydxprotocol-v4-proto-cpp/meta.yaml

```diff
@@ -0,0 +1,171 @@
+                        "liquidations.pb.h", "process_proposer_matches_events.pb.h", "mev.pb.h",
+                        "block_rate_limit_config.pb.h"],
+  'dydxprotocol/subaccounts': ["subaccount.pb.h", "asset_position.pb.h", "query.pb.h", "genesis.pb.h",
+                               "perpetual_position.pb.h"],
+  'dydxprotocol/assets': ["query.pb.h", "genesis.pb.h", "tx.pb.h", "asset.pb.h"],
+  'dydxprotocol/perpetuals': ["perpetual.pb.h", "query.pb.h", "genesis.pb.h", "tx.pb.h", "params.pb.h"],
+  'dydxprotocol/feetiers': ["query.pb.h", "genesis.pb.h", "tx.pb.h", "params.pb.h"],
+  'dydxprotocol/blocktime': ["query.pb.h", "genesis.pb.h", "tx.pb.h", "blocktime.pb.h", "params.pb.h"],
+  'dydxprotocol/rewards': ["reward_share.pb.h", "query.pb.h", "genesis.pb.h", "tx.pb.h", "params.pb.h"],
+  'dydxprotocol/sending': ["query.pb.h", "genesis.pb.h", "tx.pb.h", "transfer.pb.h"],
+  'dydxprotocol/prices': ["query.pb.h", "genesis.pb.h", "tx.pb.h", "market_param.pb.h", "market_price.pb.h"],
+  'dydxprotocol/epochs': ["query.pb.h", "genesis.pb.h", "epoch_info.pb.h"],
+  'dydxprotocol/indexer/shared': ["removal_reason.pb.h"],
+  'dydxprotocol/indexer/socks': ["messages.pb.h"],
+  'dydxprotocol/indexer/redis': ["redis_order.pb.h"],
+  'dydxprotocol/indexer/events': ["events.pb.h"],
+  'dydxprotocol/indexer/off_chain_updates': ["off_chain_updates.pb.h"],
+  'dydxprotocol/indexer/indexer_manager': ["event.pb.h"],
+  'dydxprotocol/indexer/protocol/v1': ["subaccount.pb.h", "clob.pb.h"],
+  'dydxprotocol/delaymsg': ["query.pb.h", "block_message_ids.pb.h", "genesis.pb.h", "tx.pb.h", "delayed_message.pb.h"],
+  'dydxprotocol/daemons/liquidation': ["liquidation.pb.h"],
+  'dydxprotocol/daemons/pricefeed': ["price_feed.pb.h"],
+  'dydxprotocol/daemons/bridge': ["bridge.pb.h"],
+  'dydxprotocol/stats': ["query.pb.h", "genesis.pb.h", "tx.pb.h", "stats.pb.h", "params.pb.h"],
+  'dydxprotocol/bridge': ["query.pb.h", "genesis.pb.h", "tx.pb.h", "bridge_event.pb.h", "params.pb.h",
+                          "bridge_event_info.pb.h"],
+  'dydxprotocol/vest': ["query.pb.h", "genesis.pb.h", "tx.pb.h", "vest_entry.pb.h"],
+} %}
+
+package:
+  name: dydx-v4-proto-cpp-split
+  version: {{ version }}
+
+source:
+  folder: all-sources
+  url: https://github.com/dydxprotocol/v4-clients/archive/refs/tags/v4-client-js@{{ version }}.tar.gz
+  sha256: eb6ffc4e53aa0a22f195092f2102fb5233c5c4dda7b11700208a1974d1428297
+  patches:
+    - patches/0001-build-proto-only.patch
+    - patches/0002-remove-unused-packages.patch
+    - patches/0003-add-sharedlib-install.patch
+
+build:
+  number: 0
+  # Reuiqres GCC >= 7
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}  # [not win]
+    - {{ compiler('cxx') }}  # [not win]
+    - {{ stdlib("c") }}
+    # Using experimental m2w64 toolchain for now
+    - mingw-w64-ucrt-x86_64-gcc  # [win and x86_64]
```

Comment:
These should be available using normal terms now. At least there is a UCRT migration that uses these compiler but the standard `compiler()` functions

Reply:
You lost me. I need this for gcc>7, the `blst` recipe that I am working on still show `5.3.0` for` compiler('m2w64_c')`

Reply:
This is the migration I'm referring to https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/main/recipe/migrations/r-base44_and_m2w64-ucrt.yaml

Reply:
`compiler('m2w64_c')` still installs gcc 5.3

Reply:
Ok, got it with the `conda_build_config.yaml`